Hangman/words/gr.json

```diff
@@ -1,5 +1,96 @@
 {
-	"words": ["javascript", "java", "python", "typescript", "κρεμάλα", "προγραμματιστής", "developer", "coder", "github", "google", "τερματικό", "μηχανή", "scripting", "διακομιστής", "πρόγραμμα", "σειρά", "bytecode", "εντολή", "λειτουργία", "κατασκευαστής", "διαφωνίες", "οπτικό", "μεταβλητή", "πηγή", "εκδότης", "microsoft", "βιβλιοθήκη", "εξάρτηση", "εκδοχή", "δοκιμαστικός", "πακέτο", "έγγραφα", "conditional", "επαναλήψεις", "βάση δεδομένων", "hacker", "δυαδικό", "bitwise", "δεδομένα", "αλγόριθμοι", "πελάτης", "localhost", "αντικείμενο", "τάξη", "πεδία", "ανάπτυξη", "boolean", "μέθοδοι", "παραγοντικός", "επιστροφή", "επανάληψη", "ενσάρκωση", "διακήρυξη", "τύπος", "στοίβα", "σωρός", "stackoverflow", "indentation", "regex", "μνήμη", "macros", "πληκτρολόγιο", "αυτοματισμός", "φάκελοι", "λεξικό", "symbolon", "κτίστης", "cryptex", "", "αριστούργημα", "πυραμίδα", "τελετουργικά", "νοητικός", "pythagorus", "ειλικρινής", "ναός", "πάπυρος", "εβραϊκά", "λατινικό", "μήλο", "βωμός", "crucifixion", "shaman", "darkness", "κρανίο", "vine", "ψυχή", "σύμβολα", "βιβλίο"],
+	"words": [
+		"javascript", 
+		"java", 
+		"python", 
+		"typescript", 
+		"κρεμάλα", 
+		"προγραμματιστής", 
+		"developer", 
+		"coder", 
+		"github", 
+		"google", 
+		"τερματικό", 
+		"μηχανή", 
+		"scripting", 
+		"διακομιστής", 
+		"πρόγραμμα", 
+		"σειρά", 
+		"bytecode", 
+		"εντολή", 
+		"λειτουργία", 
+		"κατασκευαστής", 
+		"διαφωνίες", 
+		"οπτικό", 
+		"μεταβλητή", 
+		"πηγή", 
+		"εκδότης", 
+		"microsoft", 
+		"βιβλιοθήκη", 
+		"εξάρτηση", 
+		"εκδοχή", 
+		"δοκιμαστικός", 
+		"πακέτο", 
+		"έγγραφα", 
+		"conditional", 
+		"επαναλήψεις", 
+		"βάση δεδομένων", 
+		"hacker", 
+		"δυαδικό", 
+		"bitwise", 
+		"δεδομένα", 
+		"αλγόριθμοι", 
+		"πελάτης", 
+		"localhost", 
+		"αντικείμενο", 
+		"τάξη", 
+		"πεδία", 
+		"ανάπτυξη", 
+		"boolean", 
+		"μέθοδοι",
+		"παραγοντικός",
+		"επιστροφή", 
+		"επανάληψη", 
+		"ενσάρκωση", 
+		"διακήρυξη", 
+		"τύπος", 
+		"στοίβα", 
+		"σωρός", 
+		"stackoverflow", 
+		"indentation",
+		"regex", 
+		"μνήμη",
+		"macros",
+		"πληκτρολόγιο", 
+		"αυτοματισμός", 
+		"φάκελοι", 
+		"λεξικό", "symbolon",
+		"κτίστης", 
+		"cryptex",
+		"", 
+		"αριστούργημα",
+		"πυραμίδα",
+		"τελετουργικά", 
+		"νοητικός", 
+		"pythagorus", 
```

Comment:
There should be transltions for pythagorus because he was a greek philospher. And aren't there any translation for other English words here like indentation, symbolon, cryptex?

Reply:
I will get to change those as well. Pythagoras is the right english spelling, in greek is Πυθαγόρας.